### PR TITLE
pin version of recent breaking updates in dependencies

### PR DESCRIPTION
**Pull request by kevinrue**

### CODE REVIEW

**Comment by kevinrue** [comment]
@bio-la 
It's ugly but it works

**Comment by kevinrue** [comment]
Sorry: actually that means that we have a better idea of version requirements to define in the toml file, so that panpipes itself requires the correct version rather than users messing around their conda environment like I do in the GitHub action.

**Comment by kevinrue** [comment]
Looks like I nailed the pyproject.toml file.

Ready to merge as far as I'm concerned. Let me know if I missed anything!

**Comment by bio-la** [comment]
ah, matplotlib

**Review by bio-la — APPROVED** [review]

File: pipeline_env.yaml [hunk 1]
```diff
@@ -16,3 +16,13 @@ dependencies:
   - pip:
```
[the remark this answers is not in the document]
Comment (kevinrue) [reply pipeline_env.yaml:16]: I've removed the pins from the yml

**Pull request merged**